src/App.tsx

```diff
@@ -15,71 +16,100 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Props = {};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+interface State {
+  selectedGood: string | null,
+}
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  selectGood = (good: string) => {
+    this.setState({ selectedGood: good });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  unselectGood = () => {
+    this.setState({ selectedGood: null });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        <h1 className={
+          selectedGood === null
+            ? 'title'
+            : 'title is-flex is-align-items-center'
+        }
+        >
+          {selectedGood === null
```

Comment:
You don't need to explicitly compare it to null, undefined, empty string etc
```suggestion
          {!selectedGood
```